html/js/desktop.js

```diff
@@ -1304,9 +1304,9 @@ function Desktop(elements) {
     var prevent = function (ev) {
         ev.preventDefault()
     }
+
     $('body')[0].addEventListener('gesturestart', prevent)
     $('body')[0].addEventListener('gesturechange', prevent)
-    $('body')[0].addEventListener('touchmove', prevent)
```

Comment:
This is right pain. Should use {passive: false} to make it actually work and then older browser will mis interpret that as another option.. Chrome automatically assume ignore, but not firefox. Both throw warnings if don;t use passive:false  unless use `touch-action` in css, which I did